pkl/AppConfig.pkl

```diff
@@ -33,3 +33,9 @@ port: Port = 5051
 
 /// Redis settings for this application
 redis: RedisConfig
+
+/// Format of the output
+outputFormat: ("toml"|"json"|"yaml"|"xml")?
+
+/// Whether or not the configuration should be pretty printed
+prettyPrint: Boolean = true
```

Comment:
Two comments here:

1. This seems a little too close to what Pkl also does, and might be confusing to users. It'd be good if the logic being configured here was more distant from Pkl's domain.
2. String literal unions are best expressed as a typealias. This means that we turn it into a Go "enum"

How about let's use log levels as a new config option? That's generally relevant to web services, which this example app is emulating. Something like:

```suggestion
/// The level of logging for the application.
///
/// - "error": Log only error level messages
/// - "warning": Log error and warning messages
/// - "info": Log all messages
logLevel: LogLevel

typealias LogLevel = "error"|"warning"|"info"
```

To get this to work, you'd want to also hook this up to a logger of some sort. Maybe implemented in-repo? Or you can use [slog](https://pkg.go.dev/log/slog) maybe

Reply:
> How about let's use log levels as a new config option?

Sounds good, I'll rework this PR to use log levels instead of output formats.